Stripe/UI/STPPaymentCardTextField.m

```diff
@@ -620,6 +653,7 @@ - (void)updateImageForFieldType:(STPCardFieldType)fieldType {
         
         [self.brandImageView.layer addAnimation:transition forKey:nil];
     }
+    [self setNeedsLayout];
```

Comment:
This can probably be moved into the `if (image != self.brandImageView.image) {` check above to avoid the occasional unnecessary layout.